_data/tools.yml

```diff
@@ -1697,3 +1697,15 @@
       url: https://docs.getzingdata.com/docs/setting-up-a-data-source/trino_setup/
     - urltext: Zing Data with Trino tutorial
       url: https://docs.starburst.io/data-consumer/clients/zingdata.html
+- name: Trino Guru
+  anchor: trino-guru
+  category: add-on
+  logo: /assets/images/logos/gurubase_logo.png
+  logosmall: /assets/images/logos/gurubase_logo_small.png
+  description: |
+    Trino Guru is a Trino-focused AI to answer your questions. 
+    It uses the data from the Trino repository and data from the official
+    Trino documentation to generate answers by leveraging the LLM. 
```

Comment:
```suggestion
   documentation to generate answers by leveraging the LLM. 
```

Reply:
I changed the entire text and am open to feedback or review.